release/DOCKERHUB.md

```diff
@@ -23,7 +23,7 @@ you are testing to ensure compatibility with configurations and scripts.
 
 ```bash
 git pull
-git checkout 0.0.3
+git checkout x.x.x
```

Comment:
Why have you gone with x.x.x? Probably should be 0.0.5

Reply:
It's a documentation file, seems a bit unnecessary to have to increment this each time. It also says above "Checkout the tag of the release you are testing" which the person checking out the release should know (especially from looking at https://github.com/nhsconnect/integration-adaptor-lab-results/releases).
But happy to change it back if it's an issue

Reply:
Let's put 0.0.5 back in please, just to be consistent with all the other images we've released